<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>271</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="nombreTienda">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>15</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailTienda">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:1em;</string>
     </property>
     <property name="maxLength">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="direccionTienda">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="telefonoTienda">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>Email: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>Nombre: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>Telefono:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>Direccion:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>Tienda</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listaProductos">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>281</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid black;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnModificarProducto">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Modificar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminarProducto">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>250</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>Cargar Archivo Binario</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cargarArchivoBinario">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Cargar</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>Guardar Archivo Binario</string>
     </property>
    </widget>
    <widget class="QPushButton" name="guardarArchivoBinario">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnAgregarProducto">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>listaProductos</zorder>
   <zorder>btnModificarProducto</zorder>
   <zorder>btnEliminarProducto</zorder>
   <zorder>frame_3</zorder>
   <zorder>btnAgregarProducto</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
